com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1229,7 +1229,16 @@ public override void Initialize(NetworkManager networkManager = null)
             // Bump the reliable window size to its maximum size of 64. Since NGO makes heavy use of
             // reliable delivery, we're better off with the increased window size compared to the
             // extra 4 bytes of header that this costs us.
-            m_NetworkSettings.WithReliableStageParameters(windowSize: 64);
+            //
+            // We also increase the maximum resend timeout since the default one in UTP is very
```

Comment:
Nitpicking , As the second argument of the function this comment seems to refer will only be applied if using UTP V 2.0+, should we add a mention in the comment that it will only be relevant for such versions ?